### ISSUE TITLE
Chapter 4 - Webpack resolve.extensions

### ISSUE THREAD

**Issue by fbertone** [verekia/js-stack-from-scratch#167]
### Chapter: 4 Webpack, React, and Hot Module Replacement - Webpack section

In webpack.config.babel.js explanation (line 81), you say: **We also want these two extensions to resolve**.
I think that's unclear / misleading.

I would rephrase it following the [documentation](https://webpack.github.io/docs/configuration.html#resolve-extensions) to something like: 
We also want these two extensions to be used to resolve modules.

Maybe also add a few more words on module resolution?

**Comment by verekia** [comment]
Fixed, thank you!
